web/src/components/metrics.tsx

```diff
@@ -78,7 +77,6 @@ import { LoadingInline } from './console/utils/status-box';
 
 import { useBoolean } from './hooks/useBoolean';
 import KebabDropdown from './kebab-dropdown';
-import IntervalDropdown from './poll-interval-dropdown';
```

Comment:
Did we remove this file or is still used elsewhere? If so, would that cause the same issue in other places?

Reply:
This file is still used on the Dashboard page. I'll remove <IntervalDropdown> when I start working on the next bug [OCPBUGS-42845](https://issues.redhat.com/browse/OCPBUGS-42845).